R/admiral-package.R

```diff
@@ -28,5 +28,5 @@
 #' @importFrom hms as_hms
 #' @importFrom lifecycle deprecate_warn deprecated deprecate_stop
 #' @importFrom stats setNames
-#' @importFrom cli cli_abort cli_div
+#' @importFrom dplyr lag
```

Comment:
can you move this up with the other dplyr functions.

also i don't think the cli stuff should get removed?  that is used throughout afaik??

Reply:
> can you move this up with the other dplyr functions.
 Yeah, I'll move lag with other dplyr package.

> also i don't think the cli stuff should get removed? that is used throughout afaik??
My bad, I thought cli_abort is being moved up with other cli packages so, removed it by mistake. Will correct that. 
![image](https://github.com/pharmaverse/admiral/assets/134052893/0cc9b469-5767-4959-8b1b-922f3fa33dd7)

Reply:
Just a thought- Shouldn't cli_div also be moved up with other cli packages mentioned in row 4 to maintain consistency and cli_abort is being mentioned twice (I am not aware if this has been done purposely)

Reply:
Yes, we should have only one entry per package.

@bms63 , should we order the packages alphabetically and within each package order the function alphabetically?

Reply:
It would make it much easier to read!  I'll make a separate issue to clean up the `admiral-packages.R` file